Files containing unused imports

```console
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/business_cycle.py:38:1: F401 'numpy as np' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/business_cycle.py:39:1: F401 'scipy.stats as st' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/commod_price.py:95:1: F401 'scipy.optimize.minimize_scalar' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/eigen_I.py:49:1: F401 'matplotlib.cm' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/eigen_I.py:51:1: F401 'mpl_toolkits.mplot3d.Axes3D' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/eigen_II.py:36:1: F401 'matplotlib.pyplot as plt' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/geom_series.py:59:1: F401 'sympy.latex' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/geom_series.py:61:1: F401 'mpl_toolkits.mplot3d.Axes3D' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/heavy_tails.py:33:1: F401 'quantecon as qe' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/heavy_tails.py:36:1: F401 'pandas_datareader.data as web' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/input_output.py:28:1: F401 'pandas as pd' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/lln_clt.py:37:1: F401 'random' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/long_run_growth.py:42:1: F401 'os' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/long_run_growth.py:43:1: F401 'matplotlib as mpl' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/long_run_growth.py:48:1: F401 'matplotlib.lines.Line2D' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/markov_chains_I.py:59:1: F401 'itertools.cycle' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/markov_chains_II.py:57:1: F401 'networkx as nx' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/markov_chains_II.py:58:1: F401 'matplotlib.cm' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/markov_chains_II.py:59:1: F401 'matplotlib as mpl' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/mle.py:17:1: F401 'scipy.stats.norm' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/mle.py:22:1: F401 'math.log' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/networks.py:59:1: F401 'quantecon_book_networks' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/olg.py:68:1: F401 'functools.partial' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/supply_demand_heterogeneity.py:40:1: F401 'pandas as pd' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/supply_demand_heterogeneity.py:41:1: F401 'matplotlib.pyplot as plt' imported but unused
/Users/thebigbool/repos/lecture-python-intro/__test_code_quality_tmp/supply_demand_multiple_goods.py:33:1: F401 'pandas as pd' imported but unused
```

Found them while working on #234

--- a/lectures/business_cycle.md
+++ b/lectures/business_cycle.md
@@ -38,8 +38,6 @@ We use the following imports
 ```{code-cell} ipython3
 import matplotlib.pyplot as plt
 import pandas as pd
-import numpy as np
-import scipy.stats as st
 import datetime
 import wbgapi as wb
 import pandas_datareader.data as web

--- a/lectures/commod_price.md
+++ b/lectures/commod_price.md
@@ -50,7 +50,7 @@ import numpy as np
 import yfinance as yf
 import matplotlib.pyplot as plt
 from scipy.interpolate import interp1d
-from scipy.optimize import minimize_scalar, brentq
+from scipy.optimize import brentq
 from scipy.stats import beta
 ```
 

--- a/lectures/eigen_II.md
+++ b/lectures/eigen_II.md
@@ -35,7 +35,6 @@ Then we will explore the Perron-Frobenius Theorem and connect it to applications
 We will use the following imports:
 
 ```{code-cell} ipython3
-import matplotlib.pyplot as plt
 import numpy as np
 from numpy.linalg import eig
 import scipy as sp

--- a/lectures/geom_series.md
+++ b/lectures/geom_series.md
@@ -56,9 +56,8 @@ import matplotlib.pyplot as plt
 plt.rcParams["figure.figsize"] = (11, 5)  #set default figure size
 import numpy as np
 import sympy as sym
-from sympy import init_printing, latex
+from sympy import init_printing
 from matplotlib import cm
-from mpl_toolkits.mplot3d import Axes3D
 ```
 
 ## Key formulas

--- a/lectures/heavy_tails.md
+++ b/lectures/heavy_tails.md
@@ -23,7 +23,7 @@ In addition to what's in Anaconda, this lecture will need the following librarie
 ```{code-cell} ipython3
 :tags: [hide-output]
 
-!pip install --upgrade yfinance quantecon pandas_datareader interpolation
+!pip install --upgrade yfinance pandas_datareader interpolation
 ```
 
 We use the following imports.
@@ -31,10 +31,8 @@ We use the following imports.
 ```{code-cell} ipython3
 import matplotlib.pyplot as plt
 import numpy as np
-import quantecon as qe
 import yfinance as yf
 import pandas as pd
-import pandas_datareader.data as web
 import statsmodels.api as sm
 
 from interpolation import interp

--- a/lectures/inflation_history.md
+++ b/lectures/inflation_history.md
@@ -60,7 +60,6 @@ import numpy as np
 import pandas as pd
 import matplotlib.pyplot as plt
 import matplotlib.dates as mdates
-import datetime
 ```
 
 ```{code-cell} ipython3

--- a/lectures/input_output.md
+++ b/lectures/input_output.md
@@ -27,7 +27,6 @@ This lecture requires the following imports and installs before we proceed.
 
 ```{code-cell} ipython3
 import numpy as np
-import pandas as pd
 import networkx as nx
 import matplotlib.pyplot as plt
 import quantecon_book_networks

--- a/lectures/lln_clt.md
+++ b/lectures/lln_clt.md
@@ -34,7 +34,6 @@ We'll need the following imports:
 
 ```{code-cell} ipython3
 import matplotlib.pyplot as plt
-import random
 import numpy as np
 import scipy.stats as st
 ```

--- a/lectures/markov_chains_I.md
+++ b/lectures/markov_chains_I.md
@@ -58,7 +58,6 @@ import numpy as np
 import networkx as nx
 from matplotlib import cm
 import matplotlib as mpl
-from itertools import cycle
 ```
 
 ## Definitions and examples

--- a/lectures/markov_chains_II.md
+++ b/lectures/markov_chains_II.md
@@ -56,9 +56,6 @@ import matplotlib.pyplot as plt
 plt.rcParams["figure.figsize"] = (11, 5)  # set default figure size
 import quantecon as qe
 import numpy as np
-import networkx as nx
-from matplotlib import cm
-import matplotlib as mpl
 ```
 
 (mc_irreducible)=

--- a/lectures/mle.md
+++ b/lectures/mle.md
@@ -14,12 +14,12 @@ kernelspec:
 # Maximum Likelihood Estimation
 
 ```{code-cell} ipython3
-from scipy.stats import lognorm, pareto, expon, norm
+from scipy.stats import lognorm, pareto, expon
 import numpy as np
 from scipy.integrate import quad
 import matplotlib.pyplot as plt
 import pandas as pd
-from math import exp, log
+from math import exp
 ```
 
 ## Introduction

--- a/lectures/networks.md
+++ b/lectures/networks.md
@@ -57,7 +57,6 @@ import pandas as pd
 import quantecon as qe
 
 import matplotlib.cm as cm
-import quantecon_book_networks
 import quantecon_book_networks.input_output as qbn_io
 import quantecon_book_networks.data as qbn_data
 

--- a/lectures/olg.md
+++ b/lectures/olg.md
@@ -65,7 +65,6 @@ Let's start with some imports.
 import numpy as np
 from scipy import optimize
 from collections import namedtuple
-from functools import partial
 import matplotlib.pyplot as plt
 ```
 

--- a/lectures/supply_demand_heterogeneity.md
+++ b/lectures/supply_demand_heterogeneity.md
@@ -37,8 +37,6 @@ Here are some imports:
 
 ```{code-cell} ipython3
 import numpy as np
-import pandas as pd
-import matplotlib.pyplot as plt
 from scipy.linalg import inv
 ```
 

--- a/lectures/supply_demand_multiple_goods.md
+++ b/lectures/supply_demand_multiple_goods.md
@@ -58,7 +58,6 @@ As usual, we start by importing some Python modules.
 ```{code-cell} ipython3
 # import some packages
 import numpy as np
-import pandas as pd
 import matplotlib.pyplot as plt
 from scipy.linalg import inv
 ```

